.storybook/preview.js

```diff
@@ -0,0 +1,11 @@
+import "../dist/js/uswds.min.js";
+
```

Comment:
It'd be best to add the main SCSS entry point here via:
`import "../src/stylesheets/uswds.scss";`

However this would require using the SCSS and PostCSS addons. The latter hasn't been updated and is stuck on `2.0.0`. This conflicts with the newest versions of Autoprefixer, even when forcing the latest PostCSS in Storybook.

When this is updated we should try again:
```js
module.exports = {
  core: {
    builder: "webpack5",
  },
  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
  addons: [
    {
      name: "@storybook/preset-scss",
      options: {
        cssLoaderOptions: {
          sourceMap: true,
          esModule: false,
        },
        sassLoaderOptions: {
          sourceMap: true,
        },
      },
    },
    {
      name: "@storybook/addon-postcss",
      options: {
        postcssLoaderOptions: {
          implementation: require("postcss"),
        },
      },
    },
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@storybook/addon-a11y",
  ],
  ...
```